sats-dna/src/main/kotlin/com/sats/dna/components/progressbar/SatsCircularSteppedProgressIndicator.kt

```diff
@@ -107,6 +120,7 @@ private fun GroupCircle(group: SteppingProgressGroup, groupNumber: Int, modifier
 @Immutable
 data class SteppingProgress(
     val groups: List<SteppingProgressGroup>,
+    val isFailed: Boolean,
```

Comment:
Idk if you guys have a better name suggestion for this? 😅

Reply:
It could probably also get a default = false, was just nice to require it when implementing its usage in the member app

Reply:
I think it works, straightforward👍